src/features/buy-now/BuyNowForm/FormSteps/Details/Details.tsx

```diff
@@ -55,3 +71,24 @@ export const Details: FC<DetailsProps> = ({
 		</>
 	);
 };
+
+/*******************
+ * TextContent
+ *******************/
+
+interface TextContentProps {
+	tokenBalance: string;
+	isLoading: boolean;
+	isOwnedByUser: boolean;
+}
+
+const TextContent = ({
+	tokenBalance,
+	isLoading,
+	isOwnedByUser,
+}: TextContentProps) =>
+	!isOwnedByUser ? (
+		<FormUserBalance tokenBalance={tokenBalance} isLoading={isLoading} />
+	) : (
+		<>You already own this NFT</>
```

Comment:
Added a quick fix up (for now) so `Buy Now` button is displayed to the user. Previously this was being hidden if the user owns the domain. This was confusing when `Floor Price` metrics were being displayed. A user will now open the modal and see this message and button will be disabled. We require a `disabled` variation of button in `zUI`.

Reply:
eidt - the Buy Now button was being hidden previously as we have been unable to disable it, meaning a user could have gone further in the Buy Now modal flow if they were able to select Buy Now from the subdomain table.

Reply:
Nice!